### PR TITLE
Feature/table tab handler

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: addon/utils/plugins/table/tab-input-plugin.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+ *
+ * @class TableBackspacePlugin
+ * @module plugins/table
+ */
+export default class TableTabInputPlugin implements TabInputPlugin {
+  label = 'backspace plugin for handling table nodes'
+
+  guidanceForManipulation(manipulation: Manipulation, editor: RawEditor) : ManipulationGuidance | null {
+    const selection = editor.selection;
+    if(selection.isInTable) {
```
Comment (nvdk) [inline addon/utils/plugins/table/tab-input-plugin.ts:17]: this is always truthful as it has the string "disabled" as value when disabled. As a result the tab handler executor will throw an error every time the plugin is called outside of a table

**Review by nvdk — CHANGES_REQUESTED** [review]
This was not properly tested outside of a table, it breaks the tab handler outside of tables.

**Comment by nvdk** [comment]
perhaps consider allowing the tab handler to jump out of the table when you press tab in the last cell?

**Comment by lagartoverde** [comment]
Now the problem I see with allowing the table to go outside is that we cannot go inside, so it feels like the navigation is broken, maybe the solution is to fix the tab handler outside of the table to jump to the first cell when it encounters a table. I have uploaded the code so you can test what it happens

**Review by nvdk — APPROVED** [review]
Looks ok to me, in firefox at least it is jumping in the table as well. though perhaps not to where we want it to jump? This could be a further improvement.

**Comment by lagartoverde** [comment]
In firefox it looks like it's jumping inside the table at the correct position but it's not if you press tab again it kinda breaks but I agree the objective of the task was to provide navigation inside the table we can fix that later on another task

**Comment by nvdk** [comment]
@lagartoverde  can you resolve the merge conflicts and merge?

**Pull request merged**